ethereum/networks/src/main/java/tech/pegasys/teku/networks/Eth2NetworkConfiguration.java

```diff
@@ -350,6 +353,16 @@ public Builder applyToledoNetworkDefaults() {
               "enr:-LK4QAhU5smiLgU0AgrdFv8eCKmDPCBkXCMCIy8Aktaci5qvCYOsW98xVqJS6OoPWt4Sz_YoTdLQBWxd-RZ756vmGPMBh2F0dG5ldHOIAAAAAAAAAACEZXRoMpCXm69nAHAe0P__________gmlkgnY0gmlwhDTTDL2Jc2VjcDI1NmsxoQOmSJ0mKsQjab7Zralm1Hi0AEReZ2SEqYdKoOPmoA98DoN0Y3CCIyiDdWRwgiMo");
     }
 
+    public Builder applyPraterNetworkDefaults() {
+      return reset()
+          .constants(PRATER.constantsName())
+          .startupTimeoutSeconds(120)
+          .eth1DepositContractAddress("0xff50ed3d0ec03aC01D4C79aAd74928BFF48a7b2b")
+          .eth1DepositContractDeployBlock(4367322)
+          // TODO .initialStateFromClasspath("prater-genesis.ssz")
```

Comment:
I think we should try loading this initial state from Url for prater rather than committing it to our own GitHub.  Otherwise we wind up bloating our git repo size a lot with the growing number of testnet states.

But the PR likely has to merge before that can happen.  And likely doesn't make much sense to merge this until we have a genesis state we can point to since it can't be rebuilt from the eth1 chain.